visio_ml/src/modules/compile_results.py

```diff
@@ -27,14 +30,15 @@ def get_exercise(self):
         return self.exercise
 
     def record_angle(self, angle) -> None:
+        t = time()
         for key, val in angle.items():
             self.results[key][val // 1] += 1
+            self.timestamped_angles[key].append((t, val // 1))
```

Comment:
you're storing [(0, 10), (0.1, 20), (0.3, 50),...]. But you can just store [10, 20, 50] and send self.delta to database along with this list.

Reply:
`t` is needed to later split the data by `delta` (0.5s)